base/server/healthcheck/pki/server/healthcheck/meta/connectivity.py

```diff
@@ -76,6 +76,7 @@ def check(self):
             logger.error("Internal server error %s", e)
             yield Result(self, constants.CRITICAL,
                          msg="Internal server error. Is your PKI server and LDAP up?",
+                         instance_name=ca.instance.name,
```

Comment:
Since this test is for CA subsystem, the error message probably should say "Is the CA subsystem and the LDAP database up?"

The `instance_name` parameter can also be defined like this so it's consistent for all subsystems:
```
instance_name=self.instance.name,
```